content/en/docs/v3.6/upgrades/upgrade_3_3.md

```diff
@@ -399,7 +399,7 @@ _, err := kvc.Get(ctx, "a")
 +  if s.Code() == codes.Canceled
 ```
```

Comment:
GitHub is rendering this section weirdly for me -- showing things have been changed above the fold. See image for what I mean. Let's double check that the changes reflect your intention.

<img width="1146" alt="Screenshot 2024-05-17 at 2 32 04 PM" src="https://github.com/etcd-io/website/assets/4453979/8e728ff9-facc-4e69-9a53-de628a370698">

Reply:
The code example starting on line 383 is a file diff illustrating the differences between the 3.2 and 3.3 config files. GitHub is interpreting the diff notation and highlighting the added and removed lines. GitHub is confused, since these removes/adds are not actually changes to the repo file.

Reply:
Thanks for confirming.